src/components/SearchIcon/SearchIcon.component.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const pathStyles = {
+  fill: '#fff',
+  fillOpacity: '1',
+};
+
+const svgStyles = {
+  position: 'absolute',
+  zIndex: '1',
+  left: '32px',
+};
+
+function SearchIcon() {
+  return (
+    <svg
+      style={svgStyles}
+      xmlns="http://www.w3.org/2000/svg"
+      version="1.1"
+      viewBox="-1 0 136 136.21852"
+      height="15pt"
+      width="15pt"
+    >
+      <g id="surface1">
```

Comment:
very interesting how is this icon set 😄 never saw something like this before, cool!
this can be smaller or simpler if an icon library is used, or you can download the svg, I let you this link: https://www.npmjs.com/package/react-icons